### ISSUE TITLE
have versions for rust language?

### ISSUE THREAD

**Issue by hgz07** [actorapp/actor-platform#539]
have versions for rust language?

**Comment by rockjam** [comment]
@hgz07 As you can see from language statistics on main page - we don't have rust language in our codebase. What did you mean exactly by "versions for rust language"?

**Comment by hgz07** [comment]
thanks @rockjam 
My project need a IM server system coded by rust language.

**Comment by rockjam** [comment]
@hgz07 no, we don't have it. Our server is written in scala. Well, if you want to use Actor as IM server for your project, I'm sure you can make integration via some kind of API.

**Comment by hgz07** [comment]
Thank u @rockjam ,
I want to run the Actor Messaging Server in cluster mode.
Are there some documents I need read ?
